int32 type
int32 cell_number
bool return_home

---

bool success